config/locales/en.yml

```diff
@@ -21,6 +21,9 @@ en:
       spree/order:
         payment_state: Payment State
         shipment_state: Shipment State
+    errors:
+      messages:
+        taken: "There's already an account for this email. Please login or reset your password."
```

Comment:
@sauloperez Thanks for the review! I added the model and field to the error message.